Packs/PaloAltoNetworks_IoT3rdParty/Scripts/SendAllPANWIoTVulnerabilitiesToSIEM/SendAllPANWIoTVulnerabilitiesToSIEM.py

```diff
@@ -0,0 +1,141 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+vulnerabilities_fields_map = [
+    ("ip", "dvc="),
+    ("deviceid", "dvcmac="),
+    ("name", "dvchost="),
+    ("profile", "cs1Label=Profile cs1="),
+    ("display_profile_category", "cs2Label=Category cs2="),
+    ("profile_vertical", "cs1Labe3=Profile cs3="),
+    ("vendor", "cs4Label=Vendor cs4="),
+    ("model", "cs5Label=Model cs5="),
+    ("vlan", "cs6Label=Vlan cs6="),
+    ("site_name", "cs7Label=Site cs7="),
+    ("risk_score", "cs8Label=RiskScore cs8="),
+    ("risk_level", "cs9Label=RiskLevel cs9="),
+    ("subnet", "cs10Label=Subnet cs10="),
+    ("vulnerability_name", "cs11Label=vulnerabilityName cs11="),
+    ("detected_date", "cs12Label=DetectionDate cs12="),
+    ("remediate_instruction", "cs13Label=RemediateInstructions cs13="),
+    ("remediate_checkbox", "cs14Label=RemediateCheckbox cs14="),
+    ("first_seen_date", "cs15Label=FirstSeenDate cs15="),
+    ("confidence_score", "cs16Label=ConfidenceScore cs16="),
+    ("os", "cs17Label=OsGroup cs17="),
+    ("os/firmware_version", "cs18Label=OsFirmwareVersion cs18="),
+    ("osCombined", "cs19Label=OsSupport cs19="),
+    ("OS_End_of_Support", "cs20Label=OsEndOfSupport cs20="),
+    ("Serial_Number", "cs21Label=SerialNumber cs21="),
+    ("endpoint_protection", "cs22Label=EndpointProtection cs22="),
+    ("NetworkLocation", "cs23Label=NetworkLocation cs23="),
+    ("AET", "cs24Label=AET cs24="),
+    ("DHCP", "cs25Label=DHCP cs25="),
+    ("wire_or_wireless", "cs26Label=WireOrWireless cs26="),
+    ("SMB", "cs27Label=SMB cs27="),
+    ("Switch_Port", "cs28Label=SwitchPort cs28="),
+    ("Switch_Name", "cs29Label=SwitchName cs29="),
+    ("Switch_IP", "cs30Label=SwitchIp cs30="),
+    ("services", "cs31Label=Services cs31="),
+    ("is_server", "cs32Label=IsServer cs32="),
+    ("NAC_profile", "cs33Label=NAC_Profile cs33="),
+    ("NAC_profile_source", "cs34Label=NAC_ProfileSource cs34="),
+    ("Access_Point_IP", "cs35Label=AccessPointIp cs35="),
+    ("Access_Point_Name", "cs36Label=AccessPointName cs36="),
+    ("SSID", "cs37Label=SSID cs37="),
+    ("Authentication_Method", "cs38Label=AuthMethod cs38="),
+    ("Encryption_Cipher", "cs39Label=EncryptionCipher cs39="),
+    ("AD_Username", "cs40Label=AD_Username cs40="),
+    ("AD_Domain", "cs41Label=AD_Domain cs41="),
+    ("Applications", "cs42Label=Applications cs42="),
+    ("Tags", "cs43Label=Tags cs43=")]
+
+
+SIEM_INSTANCE = "PANW IoT 3rd Party SIEM Integration Instance"
+PANW_IOT_INSTANCE = "PANW IoT 3rd Party Integration Instance"
+
+'''
+returns a status and message back to cloud
+'''
```

Comment:
please consider refactoring with a `main()` entry point, it is quite hard to track this code.

Reply:
added main()